cluster_experiments/experiment_analysis.py

```diff
@@ -316,6 +374,18 @@ def analysis_point_estimate(self, df: pd.DataFrame, verbose: bool = False) -> fl
         ).fit()
         return results_ols.params[self.treatment_col]
 
+    def analysis_standard_error(self, df: pd.DataFrame, verbose: bool = False) -> float:
+        """Returns the standard error of the analysis
+        Arguments:
+            df: dataframe containing the data to analyze
+            verbose (Optional): bool, prints the regression summary if True
+        """
+        results_ols = sm.OLS.from_formula(
+            self.formula,
+            data=df,
+        ).fit()
```

Comment:
You forgot to add the `cov_type` and `cov_kwds` so you are getting the std_err assuming independence between all samples